man/systemd-sleep.conf.xml

```diff
@@ -171,14 +172,6 @@
         uses the value of <varname>SuspendState=</varname> when suspending and the value of <varname>HibernateState=</varname> when hibernating.
         </para></listitem>
       </varlistentry>
-      <varlistentry>
-        <term><varname>HibernateDelaySec=</varname></term>
```

Comment:
This is now used again for the first sleep, so it should be retained and updated